src/app/containers/app/app.component.scss

```diff
@@ -2,15 +2,19 @@ $govuk-global-styles: true;
 $hmcts-images-path: "/assets/images/";
 
 // Width of main container
-$hmcts-page-width: 1220px;
+$hmcts-page-width: 1920px;
 
 // TODO temporary solution to extend the width of the page until is addressed through govuk-frontend"
 body .govuk-width-container, #content {
-  max-width: 1220px;
+  max-width: 1920px;
+  min-width: 1020px;
+  overflow-x:hidden;
```

Comment:
What is the `overflow-x: hidden` for? If we've got content exceeding the bounds that we just clip, that's not ideal...